src/encoded/static/components/filegallery.js

```diff
@@ -1390,14 +1390,15 @@ export function assembleGraph(files, highlightedFiles, dataset, options, loggedI
     let matchingFiles = {}; // All files that match the current assembly/annotation, keyed by file @id
     const fileQcMetrics = {}; // List of all file QC metrics indexed by file @id
     const allPipelines = {}; // List of all pipelines indexed by step @id
+    const allowNoAssembly = dataset.assay_term_name === 'RNA Bind-n-Seq';
```

Comment:
It is there any chance someone will use a different case like "rNA bind-n-seq"? I tend to make both hands of comparators lower case to take this into account.

Reply:
There’s no danger of that because assay_term_name has a controlled vocabulary, so people can only choose from a set of pre-defined options.